File: backend/IntencionPagoApi/Modelos/IntencionPago.cs
namespace IntencionPagoApi.Modelos
{
    public class IntencionPago
    {
        public int Id { get; set; }
        public string CodigoCliente { get; set; } = string.Empty;
        public string NombreCompleto { get; set; } = string.Empty;
        public string TipoIdentificacion { get; set; } = string.Empty;
        public string Identificacion { get; set; } = string.Empty;
        public string Telefono { get; set; } = string.Empty;
        public string Correo { get; set; } = string.Empty;
        public decimal MontoDisponible { get; set; }
        public string FechaRegistro { get; set; } = string.Empty;
        public string Estado { get; set; } = "Pendiente";
    }
}

File: frontend/payment-intention/src/app/modelos/intencion-pago.ts
export interface IntencionPago {
  id?: number;
  codigoCliente: string;
  nombreCompleto: string;
  tipoIdentificacion: 'CI' | 'PA';
  identificacion: string;
  telefono: string;
  correo: string;
  montoDisponible: number;
  fechaRegistro?: string;
  estado?: string;
}